File: project/module-script/src/test/resources/samples/javascript/branching.js
var score = 0;
for (var i = 0; i < 10000; i++) {
    if (i % 2 === 0) {
        score += i;
    } else if (i % 3 === 0) {
        score -= i;
    } else {
        score += 1;
    }
}
score;

File: project/module-script/src/jmh/resources/samples/fluxon/branching.fs
score = 0
for i in 0..<__ITER__ {
 if &i % 2 == 0 {
  score += &i
 } else if &i % 3 == 0 {
  score -= &i
 } else {
  score += 1
 }
}
&score
